client/my-sites/store/app/store-stats/index.js

```diff
@@ -57,32 +79,75 @@ class StoreStats extends Component {
 						'Learn valuable insights about the purchases made on your store.'
 					) }
 				/>
+
 				<StatsNavigation selectedItem="store" siteId={ siteId } slug={ slug } interval={ unit } />
-				<Chart
-					query={ orderQuery }
-					selectedDate={ endSelectedDate }
-					siteId={ siteId }
-					unit={ unit }
-					slug={ slug }
-				/>
-				<StatsPeriodNavigation
-					date={ selectedDate }
-					period={ unit }
-					url={ `/store/stats/orders/${ unit }/${ slug }` }
-				>
-					<DatePicker
-						period={ unit }
-						// this is needed to counter the +1d adjustment made in DatePicker for weeks
-						date={
-							unit === 'week'
-								? moment( selectedDate, 'YYYY-MM-DD' ).subtract( 1, 'days' ).format( 'YYYY-MM-DD' )
-								: selectedDate
-						}
-						query={ orderQuery }
-						statsType="statsOrders"
-						showQueryDate
-					/>
-				</StatsPeriodNavigation>
+
+				{ isNewFeatured ? (
+					<>
+						<StatsPeriodHeader>
+							<StatsPeriodNavigation
+								date={ selectedDate }
+								period={ unit }
+								url={ `/store/stats/orders/${ unit }/${ slug }` }
+							>
+								<DatePicker
+									period={ unit }
+									// this is needed to counter the +1d adjustment made in DatePicker for weeks
+									date={
+										unit === 'week'
+											? moment( selectedDate, 'YYYY-MM-DD' )
+													.subtract( 1, 'days' )
+													.format( 'YYYY-MM-DD' )
+											: selectedDate
```

Comment:
In the future, we will want to avoid using `moment` due to performance implications.

Reply:
Just want to add, moment.js is quite some noise in our bundle - circled are the language files only:

![image](https://user-images.githubusercontent.com/1425433/201567602-e5feb7d6-2d67-47bc-86dd-b1b1e0aed339.png)